common/web/gesture-recognizer/src/test/resources/json/receiver/basicMultitouch.json

```diff
@@ -7,232 +7,494 @@
+            "item": null
           },
           {
             "targetX": 46,
             "targetY": 57,
-            "t": 6983.000000000001
+            "t": 6983,
+            "item": null
           },
           {
             "targetX": 46,
             "targetY": 56,
-            "t": 7016
+            "t": 7016,
+            "item": null
           },
           {
             "targetX": 45,
             "targetY": 55,
-            "t": 7049
+            "t": 7049,
+            "item": null
           },
           {
             "targetX": 45,
             "targetY": 54,
-            "t": 7083
+            "t": 7083,
+            "item": null
           },
           {
             "targetX": 44,
             "targetY": 54,
-            "t": 7099
+            "t": 7099,
+            "item": null
           },
           {
             "targetX": 44,
             "targetY": 53,
-            "t": 7115
+            "t": 7115,
+            "item": null
           },
           {
             "targetX": 43,
             "targetY": 52,
-            "t": 7166
+            "t": 7166,
+            "item": null
           },
           {
             "targetX": 42,
             "targetY": 51,
-            "t": 7199
+            "t": 7199,
+            "item": null
           },
           {
             "targetX": 42,
             "targetY": 50,
-            "t": 7232
+            "t": 7232,
+            "item": null
           },
           {
             "targetX": 41,
             "targetY": 49,
-            "t": 7249.000000000001
+            "t": 7249,
+            "item": null
           },
           {
             "targetX": 41,
             "targetY": 48,
-            "t": 7265.000000000001
+            "t": 7265,
+            "item": null
           },
           {
             "targetX": 41,
             "targetY": 47,
-            "t": 7282
+            "t": 7282,
+            "item": null
           },
           {
             "targetX": 41,
             "targetY": 46,
-            "t": 7298
+            "t": 7298,
+            "item": null
           },
           {
             "targetX": 41,
             "targetY": 45,
-            "t": 7332
+            "t": 7332,
+            "item": null
           },
           {
             "targetX": 41,
             "targetY": 44,
-            "t": 7348
+            "t": 7348,
+            "item": null
           },
           {
             "targetX": 41,
             "targetY": 43,
-            "t": 7399
+            "t": 7399,
+            "item": null
           },
           {
             "targetX": 40,
             "targetY": 43,
-            "t": 7483.000000000001
+            "t": 7483,
+            "item": null
           },
           {
             "targetX": 40,
             "targetY": 42,
-            "t": 7917
+            "t": 7917,
+            "item": null
+          },
+          {
+            "targetX": 40,
+            "targetY": 42,
+            "t": 8151,
```

Comment:
These new entries correspond to timestamps where the _second_ contact point has recorded touch events; they're now needed within the gesture processing to ensure that gestures are only evaluated when _all_ updates for a timestamp have occurred, rather than some.

Fixing _other_ issues (namely, point 4 in the description) properly resulted in the need for this fix, too.

Also, the `item` field exists now, but it didn't at the time when this text fixture was originally made.  Doesn't hurt to have it, so I didn't bother removing it from the updated form.